src/sentry/models/organization.py

```diff
@@ -212,6 +212,9 @@ class flags(TypedClassBitField):
         # Enable codecov integration.
         codecov_access: bool
 
+        # Disable org-members from creating new projects
+        disable_member_project_creation: bool
```

Comment:
should this go after the bitfield default? 😬

Reply:
I'm guessing `bitfield_default` is used for the value of these fields when not explicitly assigned. Are we okay with the default being `1`, or `True` on this?

Reply:
Fair point about the default behaviour. I've amended it.

Reply:
Looking at this test: https://github.com/getsentry/sentry/blob/42c647d7a39742a205f5c37a7e718bfdb8aa0b91/tests/sentry/db/models/fields/bitfield/test_bitfield.py#L74-L97

The default indicates the bit in the bitfield to flip. In this case, with a default value of 1, only `allow_joinleave` will be set to true. The rest of the flags will be false.

Reply:
Nice reverse engineering! Definitely a bit misleading, and not intuitive.